spec/interactors/api/filter_events_spec.rb

```diff
@@ -3,9 +3,11 @@
 
 RSpec.shared_examples 'filtered by params' do
   subject(:sql) { events.sql }
+
   describe 'deleted record' do
-    it 'is filtered out by default' do
+    it 'filters cancelled and deleted events by default' do
```

Comment:
`deleted record -> it filters cancelled and deleted events by default` doesn't make much sense to me.

Perhaps modify the `describe` line above?